src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/PartialSuccessTransmissionPolicy.cs

```diff
@@ -14,9 +14,9 @@
 
     internal class PartialSuccessTransmissionPolicy : TransmissionPolicy, IDisposable
     {
-        private readonly JavaScriptSerializer serializer = new JavaScriptSerializer();
+        private TaskTimer pauseTimer = new TaskTimer { Delay = TimeSpan.FromSeconds(TransmissionPolicyHelpers.SlotDelayInSeconds) };
 
-        private TaskTimer pauseTimer = new TaskTimer { Delay = TimeSpan.FromSeconds(SlotDelayInSeconds) };
+        public int ConsecutiveErrors { get; set; }
```

Comment:
Yes, it will get longer even if they are unrelated.

---
In reply to: [64651855](https://github.com/Microsoft/ApplicationInsights-dotnet/pull/232#discussion_r64651855) [](ancestors = 64651855)